docs/release/kubernetes-upgrade.md

```diff
@@ -0,0 +1,240 @@
+# K3S Kubernetes Patch Release Process
+This document details the K3S kubernetes patch release process.
+
+# Before You Begin
+You’ll be primarily using git and go. Git can be installed via the local package manager. Make sure Go is installed and configured correctly, utilizing a “gopath”. This can be set via an environment variable called GOPATH. eg. export GOPATH=”${HOME}/go”, typically.
+
+## Clone and Setup Remotes
+Clone from upstream then add k3s-io fork and your personal fork.
+```sh
+# initial clone
+git clone --origin upstream \
+  https://github.com/kubernetes/kubernetes.git \
+  ${GOPATH}/src/github.com/kubernetes/kubernetes
+ 
+cd ${GOPATH}/src/github.com/kubernetes/kubernetes
+ 
+# add the k3s-io remote
+git remote add k3s-io https://github.com/k3s-io/kubernetes.git
+ 
+# fetch all remote branches and tags. If you receive a message saying that
+# previous tags will be "clobbered", add --force to the command below.
+git fetch --all --tags
+```
+# Rebasing and Generating Tags
+Establish a local branch for rebasing:
+
+```sh
+export GLOBAL_GIT_CONFIG_PATH=$(git config --list --show-origin --show-scope --global | awk 'NR==1{ split($2,path,":"); print path[2] }')
+export SSH_MOUNT_PATH=$(echo ${SSH_AUTH_SOCK} || echo "${HOME}/.ssh/id_rsa")
+
+# Set up your new/old versions of Kubernetes
+export OLD_K8S=<old-k8s-version>
+export NEW_K8S=<new-k8s-version>
+export OLD_K8S_CLIENT=<old-k8s-client-version>
+export NEW_K8S_CLIENT=<new-k8s-client-version>
+export OLD_K3S_VER="${$OLD_K8S}-k3s1" 
```

Comment:
double variable here

Reply:
resolving for future iteration

Reply:
This will be fixed before merge as this is an error. @nikolaishields